pkg/patch/internal/formatter.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/release/pkg/util"
+)
+
+type Formatter struct {
+	util.Log
+
+	CommandCreator CommandCreator
+	Style          string
+}
+
+const formatterScript = `
+set -euo pipefail
+pandoc \
+	--standalone \
+	--columns=10000000 \
+	--from=gfm \
+	--to=html5 \
+	--metadata=pagetitle="${TITLE}" \
+	--include-in-header=<(echo "${STYLE}") \
+	--output=-
+`
+
+func (r *Formatter) MarkdownToHTML(markdown, title string) (string, error) {
```

Comment:
This can be done with `blackfriday.Run([]byte(markdown))` and a HTML template. Is this already on your follow-up list?

Reply:
Sweet!

Yes, that is one of the things. From the top of my head:
- change the release noter
- change the formatter
- change the workspace / version querying

Non of those things should have external dependencies, shell out, ...

Reply:
SGTM :)